### PR TITLE
updating github activity

### PR DESCRIPTION
**Pull request by walkerlj0**

### CODE REVIEW

File: content/en/resources/github-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+### Part 1: Create an Issue
+Issues are super easy ways for you to make a suggestion to an OSS team, without having to ping them personally. These issues are then housed in a place that makes it easy for them to organize, review, and assign to others.
+
+* Open a repo. In this example, I will use the [Launchpad Repo at https://github.com/protocol/launchpad](https://github.com/protocol/launchpad).
+
+* Click on the 'Issues' tab to go to the page to submit Issues:
+![Issues Tab](issue-1.png)
+
+* Click on New Issue, choose an issue templata, and click the **Get started** button to open up a template
```
Comment (enoldev) [inline content/en/resources/github-guide/index.md:27]:
```suggestion
* Click on `New Issue`, choose an issue templata, and click the **Get started** button to open up a template
```
or
```suggestion
* Click on 'New Issue', choose an issue templata, and click the **Get started** button to open up a template
```

File: content/en/resources/github-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+![New Issue](issue-2.png)
+![Issue Type](issue-3.png)
+
+* You can delete all of the writing underneath the bold headers. Bold headers have two stars `**sentence text**`, and you can write anywhere beneath those bold headings. Not all fields are mandatory!
+
+![Create Issue](issue-4.png)
+
+* Click submit Issue and you can go back the the issues tab and see it on the list!
```
Comment (enoldev) [inline content/en/resources/github-guide/index.md:35]:
```suggestion
* Click `Submit new Issue` and you can go back the the issues tab and see it on the list!
```

**Review by enoldev — APPROVED** [review]

File: content/en/resources/github-guide/index.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+ ![Forks](fork-11.png)
+ ![Forks](fork-12.png)
+
+* In the Comparing Change page, choose your head repository (and `main` branch) on the right side of the comparison, the click **Create Pull Request**, adding a descriptive title and comments in the following screen.
+ ![Forks](fork-13.png)
+
+* Now when you check the [Pull Request for the Launchpad repo](https://github.com/protocol/launchpad/pulls) you can see your new Pull request, and make it easy for maintainers to explore and commit the changes!
+ ![Forks](fork-14.png)
+
+The maintainers of this repos only need to push a button to confirm the merge, making it much more likely (and easier for the maintainers) to get changes added.
```
Comment (mrodriguez3313) [inline content/en/resources/github-guide/index.md:86]:
``` suggestion
The maintainers of these repos only need to push a button to confirm the merge, making it much more likely (and easier for the maintainers) to get changes added.
```

File: content/en/resources/github-guide/index.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+weight: 80
+---
+
+{{< embed src="https://docs.google.com/presentation/d/11V_BA5A64lCY_wFsvuLO4je02zvzJwD5vf2kzONDWyc/embed" width="100%" height="600px">}}
+_[Github Deck Link](https://docs.google.com/presentation/d/11V_BA5A64lCY_wFsvuLO4je02zvzJwD5vf2kzONDWyc/edit?usp=sharing)_
+
+
+
+## Tutorial – Contributing to OSS on Github
+In this tutorial, you will learn how to use the GUI to open an issue on any repo, and fork and make a PR in the GUI.
```
[the remark this answers is not in the document]
Comment (walkerlj0) [reply content/en/resources/github-guide/index.md:17]: @mrodriguez3313 added to this new PR: https://github.com/protocol/launchpad/pull/431

File: content/en/resources/github-guide/index.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+![New Issue](issue-2.png)
+![Issue Type](issue-3.png)
+
+* You can delete all of the writing underneath the bold headers. Bold headers have two stars `**sentence text**`, and you can write anywhere beneath those bold headings. Not all fields are mandatory!
+
+![Create Issue](issue-4.png)
+
+* Click submit Issue and you can go back the the issues tab and see it on the list!
+
+Now the team who maintains that repo can see any problems you want to report, assign, and track who can see it. You will also get e-mail notifications for any responses people have to you. Using `@username` in a comment in an Issue or PR will notify a particular person.
```
[the remark this answers is not in the document]
Comment (walkerlj0) [reply content/en/resources/github-guide/index.md:37]: @mrodriguez3313 added to this new PR: https://github.com/protocol/launchpad/pull/431

File: content/en/resources/github-guide/index.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+![Issues Tab](issue-5.png)  
+
+### Part 2: Make a PR in the GUI
+If you want to take your changes to the next step, and make it as easy as the click of a button for repo maintainers to add a change that you suggest, you can fork the repo and make a PR.
```
[the remark this answers is not in the document]
Comment (walkerlj0) [reply content/en/resources/github-guide/index.md:42]: @mrodriguez3313 added to this new PR: https://github.com/protocol/launchpad/pull/431

File: content/en/resources/github-guide/index.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+### Modify an Existing File
+
+* First, go to the repo and choose the **Fork** button on the top right
+![Forks](fork-1.png)
+
+* A new fork (version with the same history) of that repo will be created under your organization or username.
+
+![Forks](fork-2.png)
+
+> Note that any time you go in and make changes, you will want to make sure that your repository is up to date with the original repo. You can do this from the main page of your repository fork with the **Sync fork** button. ![Forks](fork-3.png)
```
[the remark this answers is not in the document]
Comment (walkerlj0) [reply content/en/resources/github-guide/index.md:53]: @mrodriguez3313 added to this new PR: https://github.com/protocol/launchpad/pull/431

File: content/en/resources/github-guide/index.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+
+* Compare the directory structure with the URL of the curriculum and find a page you would like to make a small edit on. Note that the file you need to edit will either have the same name as the file in the URL (e.g. how-filecoin-works.md), or be and `index.md` inside of a directory with the same name as you see in the URL.
+
+![Forks](fork-7.png)
+
+* Click the Pencil Icon at the top of the page preview. From here you can make the edits you would like to that page.
+
+![Forks](fork-6.png)
+
+* Once you have completed your edits, you should click the **Commit changes** button, add a short description, and choose **Create a new Branch.** and click the **Propose changes** button. You'll be taken through two additional pages to create the pull request.
```
[the remark this answers is not in the document]
Comment (walkerlj0) [reply content/en/resources/github-guide/index.md:66]: @mrodriguez3313 added to this new PR: https://github.com/protocol/launchpad/pull/431

File: content/en/resources/github-guide/index.md (hunk 3 above)
Comment (walkerlj0) [reply content/en/resources/github-guide/index.md:86]: @mrodriguez3313 added to this new PR: https://github.com/protocol/launchpad/pull/431

**Pull request merged**